data/transactions/customTx.go

```diff
@@ -0,0 +1,53 @@
+package transactions
+
```

Comment:
What is this part designed for? no issue, just wanted to know.

Reply:
it's to add custom prefix for the txs in internal store that waiting to be executed in the block ender.
right now we only have "EXPIRE" and "FINALIZE" for proposals, this part is to add custom one so that they can be treated separately if needed